core/src/main/scala/com/github/tminglei/slickpg/geom/PgPostGISExtensions.scala

```diff
@@ -53,12 +53,12 @@ trait PgPostGISExtensions extends JdbcTypesComponent { driver: PostgresProfile =
         case Some(p) => om.column(GeomLibrary.LineFromEncodedPolyline, encodedPolyline.toNode, LiteralNode(p))
         case None   => om.column(GeomLibrary.LineFromEncodedPolyline, encodedPolyline.toNode)
       }
-    def makeBox[G1 <: GEOMETRY, P1, G2 <: GEOMETRY, P2, R](lowLeftPoint: Rep[P1], upRightPoint: Rep[P2])(
```

Comment:
Don't know why we needed a second geometry subtype here... Anyway scala 3 seemed to struggle with the type inference, and rather than specifying everything I decided to simplify the signature

Reply:
@hughsimpson can't remember it exactly.
I think your changes should be ok after double checked the official docs.